modules/skeletonRetriever/src/main.cpp

```diff
@@ -693,10 +693,14 @@ class Retriever : public RFModule
     /****************************************************************/
     bool update_gaze_frame()
     {
-        if (Property* p=gazePort.read(false))
+        Property* p;
+        if (p=gazePort.read(false)) 
         {
-            if (Bottle* b=p->find("depth_rgb").asList())
+            if (Bottle* b=p->find("depth_center").asList())
```

Comment:
Any rationale for this change?
If I recall it right, `depth_center` is just a geometric frame, whereas the keypoints of the skeleton are expressed in the sensor frame, which should be `depth_rgb` by configuration.

Reply:
I may need to summon @randaz81 for this one. We noticed that the skeleton was not aligned in the viewer compared to what we saw in the real world so we tried this change, but if I recall correctly it did not have any impact on what we saw in the viewer and I forgot to clean it up

Reply:
I think this is related to #320. The pipeline assumes that you're using the internal Realsense and not that mounted on top of the head.

Reply:
Yes indeed. This is also the source of some other bugs that we have encountered in the last days (e.g. finish-line detected below the floor by line-detector). I am reverting the change in the PR.